### PR TITLE
Remove drill progress

### PR DESCRIPTION
**Pull request by jeffreysilver**
Todo: update documentation to reflect new state of the world :)

### CODE REVIEW

File: stopcovid/dialog/aws_lambdas/publish_dialog_events_batches.py [hunk 1]
```diff
@@ -20,7 +19,6 @@ def handler(event, context):
         if record["dynamodb"].get("NewImage")
     ]
 
-    status.handle_dialog_event_batches(event_batches)
```
Comment (jeffreysilver) [inline stopcovid/dialog/aws_lambdas/publish_dialog_events_batches.py:23]: This lambda has been repurposed to just publish dialog event batches to kinesis

File: serverless.yml [hunk 2]
```diff
@@ -132,7 +132,7 @@ functions:
           startingPosition: LATEST
 
   updateDrillStatus:
-    handler: stopcovid/drill_progress/aws_lambdas/update_drill_status.handler
```
Comment (jeffreysilver) [inline serverless.yml:135]: This lambda has been repurposed to just publish dialog event batches to kinesis

Comment (pdsarin) [reply serverless.yml:135]: So these go to a kinesis stream that the new app consumes?

Comment (jeffreysilver) [reply serverless.yml:135]: Correct @pdsarin. Two main reasons:

1) Having only two consumers of the dynamo stream was going to be a problem during the migration
2) There isn't a great way to consume dynamo streams in python. I wrote a custom one modeled off of the kinesis consumer and it was pretty tricky, mainly due to the ephemerality of dynamo stream shards

**Pull request merged**